### PR TITLE
Add Vent Towers dataset

### PR DESCRIPTION
**Pull request by alexrichey**

### CODE REVIEW

**Comment by sf-dcp** [comment]
Is [this](https://github.com/NYCPlanning/data-engineering/actions/runs/7921893866) the run using the template from the pr?

File: dcpy/library/templates/dcp_air_quality_vent_towers.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      type: NONE
+
+  destination:
+    geometry:
+      SRS: null
+      type: NONE
+    options:
+      - "OVERWRITE=YES"
+      - "PRECISION=NO"
+    fields: [bbl]
```
Comment (sf-dcp) [inline dcpy/library/templates/dcp_air_quality_vent_towers.yml:21]: do fields get validated anywhere?

Comment (alexrichey) [reply dcpy/library/templates/dcp_air_quality_vent_towers.yml:21]: I have no idea, actually

Comment (alexrichey) [reply dcpy/library/templates/dcp_air_quality_vent_towers.yml]: Removed it

**Review by damonmcc — APPROVED** [review]

**Review by sf-dcp — APPROVED** [review]

**Pull request merged**